inventory.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Indentation! It's important on json files as well. Just makes them easier to read.